modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CppPistacheServerCodegen.java

```diff
@@ -410,9 +418,16 @@ public String escapeQuotationMark(String input) {
     public String escapeUnsafeCharacters(String input) {
         return input.replace("*/", "*_/").replace("/*", "/_*");
     }
-    
+
     @Override
     public String getTypeDeclaration(String str) {
         return toModelName(str);
     }
+
+    /**
+     * specify whether external libraries will be added during the generation 
```

Comment:
@etherealjoy I think you will need to add a proper docstring for the parameter "value" to address the following error reported by the CI:
```

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.0.1:jar (attach-javadocs) on project openapi-generator: MavenReportException: Error while generating Javadoc: Project contains Javadoc Warnings -> [Help 1]
[ERROR] 
```